neptune/new/internal/backends/hosted_file_operations.py

```diff
@@ -98,15 +97,19 @@ def upload_file_attribute(
             response_handler=_attribute_upload_response_handler,
             http_client=swagger_client.swagger_spec.http_client,
             url=url,
-            query_params={"experimentId": run_id, "attribute": attribute, "ext": ext},
+            query_params={
+                "experimentId": container_id,
```

Comment:
Add some note that it should be handled in NPT-.... if there is a task for getUploadConfig

Reply:
No need for that, we're going to have all in one refactor here.